packages/ide-extension/src/enhancement/enhancements.ts

```diff
@@ -1,10 +1,34 @@
 import { extensions } from 'vscode';
 import { isVSCodeCommand } from '@sap/guided-answers-extension-types';
-import type { HTMLEnhancement, NodeEnhancement } from '@sap/guided-answers-extension-types';
+import type { HTMLEnhancement, NodeEnhancement, IDE } from '@sap/guided-answers-extension-types';
 import { logString } from '../logger/logger';
 
 import enhancementsJson from './enhancements.json';
 
+/**
+ * Enumeration of environment variable used in AppStudio.
+ */
+enum ENV {
+    H2O_URL = 'H2O_URL'
+}
+
+/**
+ * Enumeration of ide platform types.
+ */
+enum IDE_PLATFORMS {
+    VSCODE = 'VSCODE',
+    BAS = 'BAS'
+}
+
+/**
+ * Return the user development platform.
+ *
+ * @returns - IDE type
+ */
+export function getIde(): IDE {
+    return process.env[ENV.H2O_URL] ? IDE_PLATFORMS.BAS : IDE_PLATFORMS.VSCODE;
```

Comment:
Will `process.env[ENV.H2O_URL]` return `BAS`? Because I was thinking it return SBAS and we name those in enhancements like that.

Reply:
It returns a dev space url or ```undefined```.